lmctl/cli/commands/__init__.py

```diff
@@ -25,6 +25,8 @@
 from .infrastructure_key import *
 from .intents import *
 from .processes import *
+from .permissiontype import *
```

Comment:
to follow convention, files named `permissiontype` should be named `permission_type` instead

Reply:
Done